e2e/bitrise.yml

```diff
@@ -111,7 +111,7 @@ workflows:
         # Limit running this test to only one stack to avoid parallel testing issues
         # Running E2E tests on all stacks in parallel would cause conflicting deployments in Google Play to the same app
         run_if: |-
-          {{ or (enveq "IS_LATEST_STACK_XCODE" "true") (not .IsCI) }}
+          {{ or (enveq "IS_LATEST_STACK_UBUNTU" "true") (not .IsCI) }}
```

Comment:
Can you tell me why a Ubuntu stack is used instead of an Xcode one?

Reply:
Paralely deploying from different builds caused issues (Error 400 Edit deleted) sometimes. So moved everything to ubuntu stack.